projects/story-decorator/src/StoryDecorator.ts

```diff
@@ -249,7 +252,8 @@ export class StoryDecorator extends SpectrumElement {
     protected override render(): TemplateResult {
         return html`
             <sp-theme
-                theme=${this.theme}
+                theme=${this.system}
```

Comment:
Do we still need `theme` here as this is internal consumption of the `<sp-theme>` element?

Reply:
No! Not needed we can remove